RIPS/rip-7560.md

```diff
@@ -836,12 +644,6 @@ All of it creates a lot of new risk vectors and attack surfaces.
 
 The following is a non-exhaustive list of known security considerations regarding Native Account Abstraction.
 
-### Attacks on validation-execution separation
-
-The state that exists at the end of the validation frame may be observed or modified by unrelated contracts before
-the execution frame begins.
-`Sender` contracts must take great care in making sure their code does not make any false assumptions.
-
```

Comment:
Since we remove validation-execution separation from this RIP, shouldn't we mention the DoS risk on block builders?  Without this separation, this RIP is only safe for chains that don't have a decentralized mempool (such as the single-sequencers used by most rollups), since they look at one transaction at a time and always add it to the current position in the block.  For chains that use a decentralized mempool, cross-transaction invalidations require that separation.

Reply:
Added "Denial of Service attacks on the block builder" security consideration paragraph. I tried to keep it generic and not use any rollup-specific jargon. Please tell me how does it look.